mtgjson5/classes/mtgjson_prices_v2_container.py

```diff
@@ -0,0 +1,99 @@
+"""
+MTGJSON Prices V2 Container
+
+This module defines the container class for organizing v2 price records by provider.
+"""
+
+from typing import Any, Dict, List
+
+from .mtgjson_prices_v2 import MtgjsonPricesRecordV2
+
+
+class MtgjsonPricesV2Container:
```

Comment:
I'm not certain about this class/container. A List of MTGJSONPricesV2 might be sufficient? Or a Dict of that?